wpilib/wpilib/timer.py

```diff
@@ -34,19 +34,15 @@ def getFPGATimestamp():
     @staticmethod
     def getMatchTime():
         """Return the approximate match time.
-        The FMS does not currently send the official match time to the robots.
-        This returns the time since the enable signal sent from the Driver
-        Station.
-        At the beginning of autonomous, the time is reset to 0.0 seconds.
-        At the beginning of teleop, the time is reset to +15.0 seconds.
-        If the robot is disabled, this returns 0.0 seconds.
-
-        .. warning::
```

Comment:
The warning needs this special formatting.

Reply:
better?

Reply:
Missed a colon, and it requires indenting at the level of the w.